### ISSUE TITLE
 More Input: If I use PyTorch Lightning for large model fine-tuning (LoRA) tasks, and I want to pass an additional parameter p outside the input-output pair (input, label), to be used in the Linear layer of LoRA for some special tasks during forward pass, how should I proceed?

### ISSUE THREAD

**Issue by AkaliKong** [huggingface/peft#1611]
How can I make it?
If I use PyTorch Lightning for large model fine-tuning (LoRA) tasks, and I want to pass an additional parameter p outside the input-output pair (input, label), to be used in the Linear layer of LoRA for some special tasks during forward pass, how should I proceed?

**Comment by BenjaminBossan** [comment]
Hi, this is the Hugging Face PEFT package and it has nothing to do with PyTorch Lightning. Please ask this question in their forums or create a GitHub issue on their repo.
